secadvisor/pkg/mangle_logstatus_test.go

```diff
@@ -34,18 +31,8 @@ pipeline:
     type: logstatus
 `)
 
-func ConfigFromJSON(t *testing.T, content []byte) *viper.Viper {
-	cfg := viper.New()
-	cfg.SetConfigType("yaml")
-	r := bytes.NewBuffer(content)
-	if err := cfg.ReadConfig(r); err != nil {
-		t.Fatalf("Failed to read config: %s", err)
-	}
-	return cfg
-}
-
 func getMangleLogStatus(t *testing.T) core.Mangler {
-	cfg := ConfigFromJSON(t, logStatusConfig)
+	cfg := core.ConfigFromJSON(t, logStatusConfig)
```

Comment:
where is the definition of this function in `core` ?

Reply:
added